src/core/mod.rs

```diff
@@ -159,6 +159,7 @@ pub enum StacksEpochId {
 }
 
 impl PartialOrd for StacksEpochId {
+    // Note: this comparison makes Epoch10 > Epoch21 > Epoch20. Is that the intention?
```

Comment:
Lol nope.

Reply:
That's what I figured -- the #2639 PR fixes that comparison.